src/app/parcels/form/ParcelForm.tsx

```diff
@@ -224,7 +223,6 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
     return (
         <CenterComponent>
             <StyledForm>
-                <Title>Parcel Form</Title>
```

Comment:
If we make the add parcel page a server compnent and follow the edit parcel page structure, this change shouldn't be needed I think...